src/components/Tour/Tour.less

```diff
@@ -0,0 +1,16 @@
+// Intro.js styles
+@import (css) '../../../node_modules/intro.js/introjs.css';
```

Comment:
No.  This may be part of the technical debt regarding our build tools. 

<img width="1117" alt="Screen Shot 2022-08-22 at 6 08 37 PM" src="https://user-images.githubusercontent.com/1038306/186027413-7930c331-d5d4-4a0c-b134-1b22e3949703.png">